src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -367,6 +369,11 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
   // Validation Rules
   readonly Rules = Rules
 
+  /** Is True if Jest is running the code. */
+  get isJestRunning (): boolean {
```

Comment:
This is already declared in Common Mixin.

Reply:
updated